api/src/middleware/routerTestUtils.ts

```diff
@@ -12,14 +15,26 @@ export const sessionDestroyMock = jest.fn();
 
 export const createAppAgentForRouter = (router: Router): SuperAgentTest => {
   const app = express();
+
+  const server = createServer(app);
+  const io = new Server(server, {
+    cors: {
+      origin: [findConfig('WEBAPP_ORIGIN', '')],
+      credentials: true,
+    },
+  });
+
+  const sessionMiddleware = expressSession({
```

Comment:
Why was `sessionMiddleware` extracted to a variable?

Reply:
Looking at the code again and comparing it with the server code, we didn't actually need to extract it to a new variable since `sessionMiddleware` is not being used anywhere else in the test.